front/app/api/content_builder_images/keys.ts

```diff
@@ -0,0 +1,9 @@
+import { QueryKeys } from 'utils/cl-react-query/types';
+
+const baseKey = { type: 'content_builder_images' };
+
+const contentBuilderImagesKeys = {
```

Comment:
I don't think we need these keys at all, since we're only adding content builder images and we're nowhere 'getting' them using this key